src/components/interactive-builder/draggable-question.component.tsx

```diff
@@ -89,7 +104,9 @@ const DraggableQuestion: React.FC<DraggableQuestionProps> = ({
             <Draggable />
           </IconButton>
         </div>
-        <p className={styles.questionLabel}>{question.label}</p>
+        <p className={styles.questionLabel}>
+          <MarkdownWrapper markdown={question.label || question.value} />
+        </p>
```

Comment:
Since the text shown in markdown can be quite long, unlike a question label, I don't think we should show the entirety of it here. Maybe we could just say `Markdown content`, or something along those lines?
@denniskigen thoughts?

Reply:
@NethmiRodrigo , we have come up with this:

![short-mark-down](https://github.com/user-attachments/assets/f6bdda67-426c-4257-a36d-a7a1fc9b8c3a)

If the markdown is very long, we show a small part of the markdown plus ellipsis to show that there is more content.

Reply:
Lovely! Thanks @Twiineenock!